### PR TITLE
Use Docker and deploy to Staging K8s

### PR DESCRIPTION
**Pull request by eskp**
Add Dockerfiles and compose

@ymleyva12

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+networks:
+  default:
+
+services:
+  backend:
+    # build: 
+    #   context: backend
+    #   dockerfile: Dockerfile
+    image: 068992353948.dkr.ecr.us-east-2.amazonaws.com/mips-backend-staging
```
Comment (sanbotto) [inline docker-compose.yml:11]: Can this also be a var? At least the account ID.

**Review by sanbotto — APPROVED** [review]

File: frontend/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export const environment = {
   production: true,
-  apiUrl: 'https://mips-api.makerdao.com',
+  apiUrl: 'https://mips-api-staging.makerdao.com',
```
Comment (sanbotto) [inline frontend/src/environments/environment.prod.ts:3]: Is this OK? This is prod's file.

File: frontend/helm/staging/values.yaml
[the remark this answers is not in the document]
Comment (eskp) [reply frontend/helm/staging/values.yaml:1]: Good pick up

File: docker-compose.yml (hunk 1 above)
Comment (eskp) [reply docker-compose.yml:11]: Removed

File: frontend/src/environments/environment.prod.ts (hunk 2 above)
Comment (eskp) [reply frontend/src/environments/environment.prod.ts:3]: Was for testing, removed

**Pull request merged**